Gems/ROS2/Code/Source/Camera/CameraSensorConfiguration.cpp

```diff
@@ -32,6 +32,8 @@ namespace ROS2
                         &CameraSensorConfiguration::m_verticalFieldOfViewDeg,
                         "Vertical field of view",
                         "Camera's vertical (y axis) field of view in degrees.")
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 179.99f)
```

Comment:
Could you leave a comment why this value is used and not 180?

Reply:
It is because we assert this to be less than 180 ([see the link](https://github.com/o3de/o3de-extras/blob/stabilization/2305/Gems/ROS2/Code/Source/Camera/CameraSensor.cpp#L72)) why then allow the user to set that?

Reply:
The key is why it is being limited to under 180, the underlying, fundamental reason